app/Jobs/CheckTraefikVersionForServerJob.php

```
@@ -0,0 +1,156 @@
+            // Only newer branch available (no patch update)
+            $this->storeOutdatedInfo($current, $newerBranchInfo['latest'], 'minor_upgrade', $newerBranchInfo['target']);
+        } else {
+            // Fully up to date
+            $this->server->update(['traefik_outdated_info' => null]);
+        }
+    }
+
+    /**
+     * Get information about newer branches if available.
+     */
+    private function getNewerBranchInfo(string $current, string $currentBranch): ?array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$current'. (undefined)

(UnusedFormalParameter)